internal/delivery/http/handlers/jwt.go

```diff
@@ -0,0 +1,132 @@
+package handlers
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"time"
+
+	jwt "github.com/golang-jwt/jwt/v5"
+
+	"github.com/andrsj/feedback-service/pkg/logger"
+)
+
+const (
+	defaultMinutes    = 10
+	defaultRole       = "all"
+	minutesQueryParam = "minutes"
+	roleQueryParam    = "role"
+	tokenPrefix       = "Bearer"
+)
+
+var (
+	errRoleParam    = errors.New("invalid role parameter")
+	errMinutesParam = errors.New("invalid minutes parameter")
+)
+
+func (h *Handlers) Token(w http.ResponseWriter, r *http.Request) {
+	var (
+		minutes int64
+		role    string
+		err     error
+	)
+
+	queryParams := r.URL.Query()
+
+	h.logger.Info("Hit Token endpoint", logger.M{"queryParams": queryParams})
+
+	minutes, err = checkMinutes(queryParams)
```

Comment:
```suggestion
	minutes, err := checkMinutes(queryParams)
```

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style